### ISSUE TITLE
"One line code" in Editor Type

### ISSUE THREAD

**Issue by Gustrucco** [mumuki/mumuki-laboratory#1305]

**Comment by flbulgarelli** [comment]
This exercise type technically exists, and is called `text`: https://github.com/mumuki/mumuki-laboratory/blob/master/app/views/layouts/exercise_inputs/editors/_text.html.erb. 

However, it looks like it is not actually working - you can create one exercise of that type in bibliotheca and laboratory will not reflect those changes. 

Also, the name `text` is really bad and obscure. I'd recommend renaming it to `one_liner`.

**Comment by flbulgarelli** [comment]
Also, see https://github.com/mumuki/mumuki-laboratory/issues/726

**Comment by rodri042** [comment]
@flbulgarelli what should the reset button do? clear the field?

**Comment by flbulgarelli** [comment]
Yes. It should behave exactly like the code-editor reset button.

**Comment by flbulgarelli** [comment]
The reset button should only clear the field in the UI, not in the backend. 

Also, you should use a one-linear code-mirror editor, not just a text-field

**Comment by luchotc** [comment]
@Gustrucco Do we need this feature for an specific exercise or it's just a general improvement?

**Comment by Gustrucco** [comment]
As far as I remember is a general improvement for exercises where you only need to write one line of code.

**Comment by luchotc** [comment]
We have to make sure that the reset button and format button works properly
